src/views/clusteroverview/OverviewTab/status-card/utils/hooks/useAlerts.ts

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { SimplifiedAlerts } from '@kubevirt-utils/components/AlertsCard/utils/types';
+import { createAlertKey } from '@kubevirt-utils/components/AlertsCard/utils/utils';
+import useKubevirtAlerts from '@kubevirt-utils/hooks/useKubevirtAlerts';
+import { Alert } from '@openshift-console/dynamic-plugin-sdk-internal/lib/api/common-types';
+import { labelsToParams } from '@virtualmachines/details/tabs/overview/utils/utils';
+
+import { AlertResource } from '../utils';
+
+const getAlertURL = (alert: Alert) =>
```

Comment:
please move this to utils :)

Reply:
Which utils? Top-level utils or component-level utils?

Reply:
component level should be enough for now :)